lib/generators/open_stax/rescue_from/install/install_generator.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails/generators'
+
+module OpenStax
+  module RescueFrom
+    class InstallGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def copy_templates
+        copy_file "any.html.erb", "app/views/errors/any.html.erb"
+      end
+
+      def copy_initializer
+        copy_file "rescue_from.rb", "config/initializers/rescue_from.rb"
+      end
+
+      def configure_environments
+        test_file = "test.rb"
+        test_middleware = <<-TEST
+  config.middleware.use ExceptionNotification::Rack, email: {
```

Comment:
Having trouble implementing this change because the Engine uses the Configuration for the email_prefix, which uses 'Server.nickname' which uses Rails.application.config, which isn't available yet, because the engine is required by the gem when it's required by Bundler in config/application.rb before the Rails app is configured